<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindowEditSettings</class>
 <widget class="KittySDK::ISettingsPage" name="ChatWindowEditSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message field</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="chatEditGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="minHeightLabel">
        <property name="text">
         <string>Minimum height:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="minHeightSpinBox">
          <property name="suffix">
           <string> rows</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fontGroupBox">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="4">
       <widget class="QComboBox" name="fontSizeComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="fontFamilyComboBox"/>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QCheckBox" name="antialiasingCheckBox">
        <property name="text">
         <string>Antialiasing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="fontFamilyLabel">
        <property name="text">
         <string>Family:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="fontSizeLabel">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="spellCheckerGroupBox">
     <property name="title">
      <string>Spell checker</string>
     </property>
     <layout class="QFormLayout" name="_5">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="spellCheckCheckBox">
        <property name="text">
         <string>Check spelling</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="spellCheckDictionaryLabel">
        <property name="text">
         <string>Dictionary:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="spellCheckDictionaryComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="spellCheckSuggestionsLabel">
        <property name="text">
         <string>Suggestion limit (0 - unlimited):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spellCheckSuggestionsSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KittySDK::ISettingsPage</class>
   <extends>QWidget</extends>
   <header>ISettingsPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spellCheckCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spellCheckDictionaryComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spellCheckCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spellCheckSuggestionsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
